content/blog/2016/2016-05-31-Jenkins-Usage-Statistics-Analysis-Intro.adoc

```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+title: "GSoC: Jenkins Usage Statistics Analysis Introduction"
+tags:
+-infra_statistics
+-gsoc
+-jenkins_usage
+author: payal94
+---
+
+=== About myself
+Hello, my name is Payal Priyadarshini.
+I am pursing my major in Computer Science & Engineering at the Indian Institute of Technology Kharagpur, India.
+I am very proficient in writing code in Python, C++, Java and currently getting familiar and hopefully good in Groovy too.
+
+I have internship experiences in renowed institutions like Google and VMware where I worked with some exciting
+technologies for example Knowledge Graphs, BigTable, SPARQL, RDF in Google. I am a passionate computer science student who
+is always interested in learning and looking for new challenges and technologies.That’s how I came across to Google Summer 
+of Code where I am working on some exciting data mining problems which you are going to encounter below in this blog.
+
+=== Project Overview
+Jenkins has collected anonymous jenkins usage information of more than 100,000 installations which includes set of plugins
+and their versions, release history etc. This data collection can be exploited for various data mining experiments. 
```

Comment:
I suggest using a less negatively charged word than "exploited" in this sentence. How about "utilized"?